tabs/motors.js

```diff
@@ -167,8 +174,14 @@ TABS.motors.initialize = function (callback) {
         lines.attr('d', graphHelpers.line);
     }
 
-    function update_model(val) {
-        $('.mixerPreview img').attr('src', './resources/motor_order/' + mixerList[val - 1].image + '.svg');
+    function update_model(mixer) {
```

Comment:
This seems to be unrelated to the battery configuration. Should this be in here, or should it be a separate PR?